CHANGELOG.next.asciidoc

```diff
@@ -39,6 +39,18 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 *Metricbeat*
 
 - Setting period for counter cache for Prometheus remote_write at least to 60sec {pull}38553[38553]
+- Add support of Graphite series 1.1.0+ tagging extension for statsd module. {pull}39619[39619]
+- Allow metricsets to report their status via control v2 protocol. {pull}40025[40025]
+- Remove fallback to the node limit for the `kubernetes.pod.cpu.usage.limit.pct` and `kubernetes.pod.memory.usage.limit.pct` metrics calculation
+- Add support for Kibana status metricset in v8 format {pull}40275[40275]
+- Update metrics for the vSphere Host metricset. {pull}40429[40429]
+- Added back `elasticsearch.node.stats.jvm.mem.pools.*` to the `node_stats` metricset {pull}40571[40571]
+
+*Osquerybeat*
```

Comment:
This line and the ones below seem not to have been part of the original PR I think? 🤔 : https://github.com/elastic/beats/pull/40571/files

Reply:
That's weird. I'll manually remove them.

Reply:
Fixed